homework-ajax/js/script.js

```diff
@@ -0,0 +1,25 @@
+const URL = 'https://jsonplaceholder.typicode.com/posts';
+
+const form = document.getElementById('form');
+form.addEventListener('submit', handleSubmitForm);
+
+function handleSubmitForm(event) {
+    event.preventDefault();
+    const newData = new FormData(event.target);
+    const fromForm = Object.fromEntries(newData.entries());
+
+    fetch(URL, {
+            method: "POST",
+            body: JSON.stringify(fromForm),
+            headers: {
+                'Content-Type': 'application/json',
+            }
+        })
+        .then((res) => res.json())
+        .then(data => {
+            console.log(data);
```

Comment:
Try no to just console.log data. Try to use it - build HTML, append to <BODY>, make some buttons etc.